e2e/node/resource_node.go

```diff
@@ -428,6 +458,8 @@ func resourceUpdateNode(ctx context.Context, d *schema.ResourceData, m interface
 	if d.HasChange("power_status") {
 		nodestatus := d.Get("status").(string)
 		if nodestatus == "Creating" || nodestatus == "Reinstalling" {
```

Comment:
"Creating" and "Reinstalling" is the same case as "Failed" state. Put these in a constant.